### ISSUE TITLE
Drift velocity calculation changed in ICARUS

### ISSUE THREAD

**Issue by brucehoward-physics** [SBNSoftware/sbncode#205]
With the merge of PR https://github.com/SBNSoftware/icarusalg/pull/43 in `icarusalg` ICARUS switched to an alternate description of drift velocity (`UseIcarusMicrobooneDriftModel: true`). At the field/temperature currently set in the ICARUS parameters this is a fairly small change the velocity.

Filing an issue since the same change was not made in SBND, at least yet, so that we can remember to have any necessary discussion in the future and push the fhicl change there as well.

**Comment by wesketchum** [comment]
Assigning to Mike and Filippo, though @marcodeltutto and others may also want to weigh in.

I've also targeted this for 2022B (spring release) since that's the critical juncture for joint SBND/ICARUS analysis pieces, but the sooner it can be decided/get in the better.

**Comment by varanini** [comment]
I agree with @mrmooney that the ICARUS change is both good and should have a fairly small impact on reconstruction. And should be extended to SBND.
Not in the position to comment much on the technical issues and timescale for the SBND change.

**Comment by ibsafa** [comment]
@brucehoward-physics has this issue been resolved?

**Comment by brucehoward-physics** [comment]
This is a blast from the past. I'd leave that to @mrmooney as I'm not sure if the similar change was ever made in SBND or if they had those discussions. I'll also tag @varanini and @linyan-w as the other ICARUS and SBND calibration conveners, though this GitHub Issue predates Linyan joining so definitely not expecting her to have the answer! Just wanted to include relevant folks who could discuss amongst themselves.
